.github/workflows/ruby.yml

```diff
@@ -10,11 +10,19 @@ permissions:
   contents: read
 
 jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@3fbf038d6f0d8043b914f923764c61bc2a114a77
+    with:
+      engine: all
```

Comment:
Let’s set this to just CRuby. We can add support for JRuby and TruffleRuby later. It’s going to require a little work on Quickdraw first.